docs/source/federated-types/federated-directives.mdx

```diff
@@ -657,3 +657,71 @@ If any of these requirements is not met, composition fails.
 
 </tbody>
 </table>
+
+## Security
+
+### `@authenticated`
+
+> ⚠️ **This directive is available in Apollo Federation 2.5 and later.**
+> ⚠️ **This is an [Enterprise feature](/router/enterprise-features) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).</br>
```

Comment:
Something here is [causing a syntax error in the MDX](https://app.netlify.com/sites/apollo-federation-docs/deploys/64b84529825acf0008f80831#L185). I suspect it's the `</br>` tag, but it's just a guess. Maybe try removing them here and elsewhere. We don't have any usages of them in mdx files anywhere.

Reply:
MDX supports HTML but I think Gatsby was having some issues -> I changed it to match other break tags